cob_image_flip/ros/include/cob_image_flip/image_flip.h

```diff
@@ -114,9 +114,15 @@ class ImageFlip
 
 	void disparityCallback(const stereo_msgs::DisparityImage::ConstPtr& disparity_image_msg);
 
-	void disparityConnectCB(const ros::SingleSubscriberPublisher& pub);
+    	void disparityConnectCB(const ros::SingleSubscriberPublisher& pub);
```

Comment:
Please keep the indentation as is, i.e. tabs in all these files instead of spaces.